src/libostree/ostree-repo.c

```diff
@@ -2598,6 +2598,19 @@ append_one_remote_config (OstreeRepo      *self,
   return add_remotes_from_keyfile (self, remotedata, path, error);
 }
 
+static gboolean
+min_free_space_str_validate (char *min_free_space_str)
+{
+  GRegex *percent_regex = g_regex_new ("^([0-9]|[1-9][0-9])%$" , 0,0, NULL);
+  GRegex *size_regex = g_regex_new ("^[0-9]+(G|M|T)B$", 0,0, NULL);
```

Comment:
`g_autoptr()` here - these are leaked currently.  (Also we tend to use `g_once_init...()` for regexps, see elsewhere in the code)